internal/util/util.go

```diff
@@ -92,3 +96,26 @@ func Stringify(input any) string {
 	}
 	return string(inputBytes)
 }
+
+// MibToBytes returns the bytes equivalent
+// of given no.s of MiBs (Mibi Bytes).
```

Comment:
nit: Mib can be read as Mibi `bits`. Hence we should rename it more clear. Maybe, `MegaBytesToBytes` or `MibiBytesToBytes`

Reply:
Renamed to MiBsToBytes which should do.